ci-operator/step-registry/baremetalds/e2e/ovn/bgp/pre/baremetalds-e2e-ovn-bgp-pre-commands.sh

```diff
@@ -81,4 +81,14 @@ spec:
           allowed:
             mode: filtered
 EOF
+
+CLUSTER_NETWORK="10.128.0.0/14"
+EGRESSIP_CIDR="192.169.100.0/24"
+
+iptables -t filter -I FORWARD -s ${CLUSTER_NETWORK} -i ostestbm -j ACCEPT
+iptables -t filter -I FORWARD -d ${CLUSTER_NETWORK} -o ostestbm -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
+iptables -t filter -I FORWARD -s ${EGRESSIP_CIDR} -i ostestbm -j ACCEPT
+iptables -t filter -I FORWARD -d ${EGRESSIP_CIDR} -o ostestbm -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
+iptables -t nat -I POSTROUTING -s ${CLUSTER_NETWORK} ! -d 192.168.111.1/24 -j MASQUERADE
+iptables -t nat -I POSTROUTING -s ${EGRESSIP_CIDR} ! -d 192.168.111.1/24 -j MASQUERADE
```

Comment:
Is this also a temporary hack, or something that will be documented as required, or... ?

Where is the firewall that is dropping the packets that necessitates these ACCEPT rules? Note that eventually everything will move to nftables and this will no longer work; you can't accept packets that someone else is trying to drop with nftables.

Reply:
This is specific to the metallb test environment and nothing that has to do with the cluster itself so we should be good.

Reply:
I guess a valid consideration for the hypervisor itself, I don't know exactly how libvirt is going to implement this with nftables.